Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -17,11 +17,12 @@
 from EventFilter.CastorRawToDigi.CastorDigiToRaw_cfi import *
 from EventFilter.RawDataCollector.rawDataCollector_cfi import *
 from L1Trigger.Configuration.L1TDigiToRaw_cff import *
+from EventFilter.CTPPSRawToDigi.ctppsDigiToRaw_cfi import *
 
 from Configuration.ProcessModifiers.premix_stage1_cff import premix_stage1
 
 #DigiToRaw = cms.Sequence(L1TDigiToRaw*siPixelRawData*SiStripDigiToRaw*ecalPacker*esDigiToRaw*hcalRawData*cscpacker*dtpacker*rpcpacker*rawDataCollector)
-DigiToRaw = cms.Sequence(L1TDigiToRaw*siPixelRawData*SiStripDigiToRaw*ecalPacker*esDigiToRaw*hcalRawData*cscpacker*dtpacker*rpcpacker*castorRawData*rawDataCollector)
+DigiToRaw = cms.Sequence(L1TDigiToRaw*siPixelRawData*SiStripDigiToRaw*ecalPacker*esDigiToRaw*hcalRawData*cscpacker*dtpacker*rpcpacker*ctppsRawData*castorRawData*rawDataCollector)
```

Comment:
add this with ```ctpps_2016```
see e.g. https://github.com/cms-sw/cmssw/blob/b59dc5246fb83a92d5f9cf445a6f2891d6181a32/Configuration/StandardSequences/python/RawToDigi_cff.py#L98-L102